test/Test.cpp

```diff
@@ -78,32 +78,36 @@ void input(const char *str)
     }
 }
 
-#define REQUEST(x)          input(x);INIT_BUF();cmd_char_input('\r');
-#define CMDLINE_BEGIN       "\r\x1b[2K/>"
-#define CMDLINE_ENDS        "\x1b[1D"
-#define CMDLINE(x)          CMDLINE_BEGIN x CMDLINE_ENDS
-#define CMDLINE_EMPTY       CMDLINE(" ")
-#define RESPONSE(x)         "\r\n" x "\r\n" CMDLINE_EMPTY
+#define ESCAPE(x) "\x1b" x
 
+#define DEFAULT_PROMPT      "/>"
```

Comment:
`DEFAULT_PROMPT` is not shown, only when serial get something input after `reset` button pressed.
it's just looking like below, only when typing something, "/>" will appear
```
ARM Ltd
[INFO][main]: write 'help' and press ENTER
```